.github/workflows/wheel.yml

```diff
@@ -0,0 +1,34 @@
+name: nightly
```

Comment:
I like the idea. I was just thinking of the name "nightly", as this file as of today does not really run nightly (with cron) but rather only on merge of pull requests.

(Theoretically we may get regressions even without changes in our repo, like if an action gets outdated or updated without changing major version)

Alternatives:

- Keep it as is (works for me)
- Use some other name/tag than "nightly"?
- Add a cron as well so it runs nightly as well?

Reply:
True, I  was also playing with the wording. Nightly is technically not completely correct as you correctly mentioned since it is triggered on master pushes. How about `latest`? The problem I have with `latest` is that it usually means latest release but we are not talking about a release here

Reply:
Changed trigger to schedule at midnight and therefore making it a **real nightly**